cellbase-server/src/main/java/org/opencb/cellbase/server/rest/GenericRestWSServer.java

```diff
@@ -313,14 +313,12 @@ protected Response createErrorResponse(Exception e) {
                 .build();
     }
 
+    protected Response createErrorResponse(Exception e) {
+        return createErrorResponse(e.toString());
```

Comment:
We should not lost the "logger.error" call with the exception as the last parameter. This is important because this will print the whole stacktrace

```java
// First we print the exception in Server logs
logger.error("Catch error: " + e.getMessage(), e);
```

Reply:
Yes, but that method is not going to print the whole stack trace of the exception. The "magic" happens when the Exception is the second parameter of the method. 
See https://www.javadoc.io/doc/org.slf4j/slf4j-api/1.7.32/org/slf4j/Logger.html#error(java.lang.String,%20java.lang.Throwable)